src/main/java/org/opensearch/security/ssl/util/SSLConfigConstants.java

```diff
@@ -99,9 +99,9 @@ public final class SSLConfigConstants {
 
     public static final String JDK_TLS_REJECT_CLIENT_INITIATED_RENEGOTIATION = "jdk.tls.rejectClientInitiatedRenegotiation";
 
-    private static final String[] _SECURE_SSL_PROTOCOLS = { "TLSv1.3", "TLSv1.2", "TLSv1.1" };
+    private static final String[] _SECURE_SSL_PROTOCOLS = { "TLSv1.3", "TLSv1.2" };
```

Comment:
What do you think about removing this in the defaults for all of the security plugin, [link](https://github.com/search?q=repo%3Aopensearch-project%2Fsecurity+TLSv1&type=code&p=1).  There are many additional usages.

Reply:
+1